src/scenario.jl

```diff
@@ -149,18 +151,26 @@ function run_scenarios(
     # This is also forced me to added an argument to run_model which breaks
     # encapsulation, so its quite ugly
     n_locs::Int64 = dom.coral_growth.n_locs
-    C_bins::Matrix{Float64} = bin_edges() ./ 100
     n_sizes::Int64 = dom.coral_growth.n_sizes
     n_groups::Int64 = dom.coral_growth.n_groups
     functional_groups = [
         FunctionalGroup.(
-            eachrow(C_bins[:, 1:end-1]), eachrow(C_bins[:, 2:end]), eachrow(zeros(n_groups, n_sizes))
+            eachrow(bin_edges()[:, 1:end-1]),
+            eachrow(bin_edges()[:, 2:end]),
+            eachrow(zeros(n_groups, n_sizes))
         ) for _ in 1:n_locs
     ]
 
+    loc_habitable_area = site_k_area(dom)
+    max_projected_cover = CoralBlox.max_projected_cover(
+        linear_extensions(),
+        bin_edges(),
+        loc_habitable_area
+    )
```

Comment:
Micro-performance change:

`bin_edges()` could be called once (assign to a variable), to avoid allocating three separate times.

Reply:
Doing so will address this comment too: https://github.com/open-AIMS/ADRIA.jl/pull/805/files#r1682389979